dataflow/pom.xml

```diff
@@ -23,84 +25,209 @@
   <version>1.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
-  <!--
-  The parent pom defines common style checks and testing strategies for our samples.
-  Removing or replacing it should not affect the execution of the samples in anyway.
-  -->
-  <parent>
-    <groupId>com.google.cloud.samples</groupId>
-    <artifactId>shared-configuration</artifactId>
-    <version>1.0.9</version>
-  </parent>
-
   <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <java.version>1.8</java.version>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
-    <apache_beam.version>2.9.0</apache_beam.version>
+    <beam.version>2.36.0</beam.version>
```

Comment:
I tested the pipeline with the most recent version of 2.37.0;  I propose this be updated.